src/TestSuite/Fixture/DataFixtureManager.php

```diff
@@ -0,0 +1,252 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.3.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite\Fixture;
+
+use Cake\Core\Configure;
+use Cake\Core\Exception\CakeException;
+use Cake\Datasource\ConnectionInterface;
+use Cake\Datasource\ConnectionManager;
+use Cake\TestSuite\TestCase;
+use PDOException;
+use RuntimeException;
+use UnexpectedValueException;
+
+/**
+ * Data only fixture manager
+ *
+ * Part of the data-only fixture system. Data-only fixtures assume that
+ * you have created the schema for the testsuite *before* running tests.
+ * They also assume that any mutations to your schema are reverted by you.
+ *
+ * This class implements a common interface with the Cake\TestSuite\FixtureManager so that
+ * test cases only need to interact with a single interface.
+ */
+class DataFixtureManager
```

Comment:
This sounds more like `FixtureDataManager` instead of `DataFixtureManager` from the description. 

However, I don't know if some other fixture system sets up the schema or if it's entirely outside of the code.

Reply:
For the new fixtures the schema would be managed externally to this class. I'm fine with FixtureDataManager if that resonates better with others.